src/modules/forms/plugins/RequestTransformer/RequestTransformerPlugin.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import block from '../../../../helpers/bem-cn';
+
+import Row from '../../../Layout/Row/Row';
+import Label from '../../../labels/Label';
+import Input from '../../../inputs/Input';
+import ControlBar from '../ControlBar/ControlBar';
+import HeadersSection from './HeadersSection/HeadersSection';
+import QueryStringSection from './QueryStringSection/QueryStringSection';
+
+const propTypes = {
+    className: PropTypes.string,
+    name: PropTypes.string.isRequired,
+    pluginFromValues: PropTypes.object.isRequired,
+    pluginName: PropTypes.string.isRequired,
+    handlePluginExclude: PropTypes.func.isRequired,
+};
+
+const RequestTransformerPlugin = ({ className, name, handlePluginExclude, pluginFromValues, pluginName }) => {
+    const b = block(className);
+
+    return (
+        <div className={b('section')()}>
+            <Row fullwidth>
+                <Row col>
+                    <Label>Plugin Name</Label>
+                    <Input input={{value: 'Request Transformer'}} disabled />
+                </Row>
+                <ControlBar name={`${name}.enabled`} removePlugin={() => handlePluginExclude(pluginName)} />
+            </Row>
+            <Row className={b('row')()} fullwidth>
```

Comment:
I see this repeats a lot any why to DRY it?

Reply:
@Vxplus  the same as with the next comment... will be completely removed